types/key.go

```diff
@@ -175,17 +175,9 @@ const (
 	// OSImageSLES15 is the OS Image value of a Node.
 	OSImageSLES15 string = "SUSE Linux Enterprise Server 15"
 
-	// CSIOperatorFilePrefix is the csi operator file path prefix.
-	CSIOperatorFilePrefix string = "/templates/csi-operator-"
-	// CSIOperatorFileSuffix is the csi operator file path suffix.
-	CSIOperatorFileSuffix string = ".yaml"
-	// CSIOperatorSUSE12FileSuffix is the csi operator file for suse 12 path suffix.
-	CSIOperatorSUSE12FileSuffix string = "-sles-12.yaml"
-	// CSIOperatorSUSE15FileSuffix is the csi operator file for suse 15 path suffix.
-	CSIOperatorSUSE15FileSuffix string = "-sles-15.yaml"
-	// CSIOperatorUbuntu1804FileSuffix is the csi operator file for ubuntu 18.04 path suffix.
-	CSIOperatorUbuntu1804FileSuffix string = "-ubuntu-18.04.yaml"
-
```

Comment:
Where are the above changes moved to?

Reply:
Earlier I was using different files, then thought every release will have 5 extra files. Now the OS dependent part is handled from code itself. CSI yamls are also in openebs-operator file only.